widgets/src/js/operations/ot/NodeAddOperation.js

```diff
@@ -89,6 +92,13 @@ define([
          */
         var _zIndex = zIndex;
 
+        /**
+         * Position of node on z-axis
+         * @type {number}
```

Comment:
Shouldn't this be of type boolean?
And the description might be the one from zIndex.